inst/examples/mnist/train.R

```diff
@@ -1,4 +1,12 @@
 library(tensorflow)
+library(optparse)
+
+message("Command Arguments: ", paste(commandArgs(trailingOnly = TRUE), collapse = " "))
```

Comment:
Hoping that all of this is just part of development (ideally we want to use FLAGS for all tunable parameters, even if we need to make changes to tfruns to make this work).

Reply:
@jjallaire I'm mostly trying to get this to work, but yes, I'll refactor this to use flags. Somehow, the cloudml runs seem to be missing the hyperparameter args, investigating...